src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue2953.cs

```diff
@@ -12,16 +12,12 @@ public Issue2953(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "GroupHeaderCells disappear when item is removed from a group in ListView (iOS only) ";
 
-	// [Test]
-	// [Category(UITestCategories.ListView)]
-	// [FailsOnIOS]
-	// public void Issue2953Test()
-	// {
-	// 	App.Screenshot("I am at Issue 2953");
-	// 	App.WaitForElement(q => q.Marked("Header 3"));
-	// 	App.Screenshot("I see the Header 3");
-	// 	App.Tap(q => q.Marked("btnRemove"));
-	// 	App.WaitForElement(q => q.Marked("Header 3"));
-	// 	App.Screenshot("I still see the Header 3");
-	// }
+	[Test]
+	[Category(UITestCategories.ListView)]
+	public void Issue2953Test()
+	{
+		App.WaitForElement("Header 3");
+		App.Tap("btnRemove");
+		App.WaitForElement("Header 3");
+	}
```

Comment:
Not impact in the test results etc, but could you leave the original test lines taking screenshots?
Like:
`App.Screenshot("I still see the Header 3");`